packages/compass-schema/src/components/compass-schema/compass-schema.module.less

```diff
@@ -365,6 +369,8 @@
             -moz-user-select: none;
             -webkit-user-select: none;
             -ms-user-select: none;
+            padding: 2px 4px;
+            border-radius: 4px;
```

Comment:
Not sure when these started breaking, but these changes fix them.
Before
<img width="1168" alt="Screen Shot 2022-02-16 at 7 07 52 PM" src="https://user-images.githubusercontent.com/1791149/154378772-4ca6811c-af8a-47ee-81f3-bba84e78f474.png">

After
<img width="891" alt="Screen Shot 2022-02-16 at 7 07 42 PM" src="https://user-images.githubusercontent.com/1791149/154378766-867f86ae-e748-4ca8-bb43-7f0f0a961b4e.png">